linux/base.Dockerfile

```diff
@@ -93,6 +93,7 @@ RUN apt-get update && bash ./aptinstall.sh \
   less \
   libffi-dev \
   libssl-dev \
+  libpq-dev \
```

Comment:
Is `libpq-dev` enough? We may also need `gcc`:

https://github.com/Azure/azure-cli/blob/2e87ac82207b345be80aa7051cabf5071f1b94dd/src/azure-cli-core/azure/cli/core/extension/operations.py#L216

```py
            exit_code = subprocess.call(['dpkg', '-s', 'gcc', 'libpq-dev', 'python3-dev'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```